eth-connector/src/connector.rs

```diff
@@ -10,6 +11,20 @@ pub trait Deposit {
     fn deposit(&mut self, #[serializer(borsh)] raw_proof: Proof) -> Promise;
 }
 
+#[ext_contract(ext_fee_manage)]
+pub trait FeeManagement {
+    fn get_deposit_fee_percentage(&self) -> DepositFeePercentage;
+    fn get_withdraw_fee_percentage(&self) -> WithdrawFeePercentage;
+    fn get_deposit_fee_bounds(&self) -> FeeBounds;
+    fn get_withdraw_fee_bounds(&self) -> FeeBounds;
+    fn check_fee_bounds(&self, amount: u128, is_deposit: bool) -> u128;
```

Comment:
As I understand you decided to use a different implementation depending on the flag `is_deposit`? If so, then It's a bad practice. For example, we will decide to add one more fee type later. In this case, we will break the interface.  That is bad. So I propose two ways here. 1. Create different method for every type of fee. 2. To use enum Instead of  bool: `fn check_fee_bounds(&self, amount: u128, fee_type: FeeType) -> u128;`
```rust
enum FeeType {
    Deposit,
    Withdraw,
}
```

Reply:
fixed in c7fdabf5f732a20e7488eacbdad563420a525587